src/components/Proposal/VoteBox.tsx

```diff
@@ -313,13 +334,23 @@ class VoteBox extends React.Component<IContainerProps, IState> {
             /> : ""
           }
           <div className={css.castVote}>
-            <span className={css.buttonsOnlyVoteLabel}>Vote: </span>
-            <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 1)} className={voteUpButtonClass}>
-              <img src="/assets/images/Icon/vote/for-btn-selected-w.svg"/><span> For</span>
-            </button>
-            <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 2)} className={voteDownButtonClass}>
-              <img src="/assets/images/Icon/vote/against-btn-selected-w.svg"/><span> Against</span>
-            </button>
+            {!votingDisabled ? 
+              <div>
+                <span className={css.buttonsOnlyVoteLabel}>Vote: </span>
+                <button onClick={votingDisabled ? null : this.handleClickVote.bind(this, 1)} className={voteUpButtonClass}>
```

Comment:
i cant believe this code has to be duplicated three times. this needs some serious refactoring. you dont have to do it now though